packages/model-viewer/src/features/scene-graph.ts

```diff
@@ -84,6 +86,10 @@ export const SceneGraphMixin = <T extends Constructor<ModelViewerElementBase>>(
       return this[$variants];
     }
 
+    get gltfJson() {
+      return this[$currentGLTF]?.correlatedSceneGraph.gltf;
```

Comment:
I'm using this in the editor as well. Perhaps would be better to clone? As I'm looking at it, I'm wondering if someone could do something odd by editing this...

Reply:
yeah cloning wouldn't hurt, changing the gltf object directly could probably interfere with internal representation, although it would make sense to edit and then reload the scene with the changes, but that's a different can of worms.